src/openrct2/network/Network.cpp

```diff
@@ -3776,77 +3757,55 @@ void game_command_modify_groups(
             }
         }
         break;
-        case 3:
-        { // set group name
-            NetworkGroup* group = gNetwork.GetGroupByID(groupid);
+        case ModifyGroupType::SetName:
+        {
+            NetworkGroup* group = gNetwork.GetGroupByID(groupId);
             const char* oldName = group->GetName().c_str();
-            static char newName[128];
-
-            size_t nameChunkOffset = nameChunkIndex - 1;
-            if (nameChunkIndex == 0)
-                nameChunkOffset = 2;
-            nameChunkOffset *= 12;
-            nameChunkOffset = (std::min)(nameChunkOffset, std::size(newName) - 12);
-            std::memcpy((void*)((uintptr_t)newName + (uintptr_t)nameChunkOffset + 0), edx, sizeof(uint32_t));
-            std::memcpy((void*)((uintptr_t)newName + (uintptr_t)nameChunkOffset + 4), ebp, sizeof(uint32_t));
-            std::memcpy((void*)((uintptr_t)newName + (uintptr_t)nameChunkOffset + 8), edi, sizeof(uint32_t));
-
-            if (nameChunkIndex != 0)
-            {
-                *ebx = 0;
-                return;
-            }
 
-            if (strcmp(oldName, newName) == 0)
+            if (strcmp(oldName, name.c_str()) == 0)
             {
-                *ebx = 0;
-                return;
+                return std::make_unique<GameActionResult>();
             }
 
-            if (newName[0] == 0)
+            if (name.empty())
             {
-                gGameCommandErrorTitle = STR_CANT_RENAME_GROUP;
-                gGameCommandErrorText = STR_INVALID_GROUP_NAME;
-                *ebx = MONEY32_UNDEFINED;
-                return;
+                return std::make_unique<GameActionResult>(
+                    GA_ERROR::INVALID_PARAMETERS, STR_CANT_RENAME_GROUP, STR_INVALID_GROUP_NAME);
             }
 
-            if (*ebx & GAME_COMMAND_FLAG_APPLY)
+            if (isExecuting)
             {
                 if (group)
                 {
                     // Log edit player group name event
```

Comment:
I think the game action logging should cover this already so you can remove this block

Reply:
Does that go for all of the logging in all of this command.

Reply:
If doing this over game actions is the only way now then I think this can be dropped as the game actions logging should provide enough details given by the parameters, but I think its safe to ditch it.